styles/_tablet.scss

```diff
@@ -0,0 +1,68 @@
+
+
+.row {
+  flex-direction: column;
+}
+
+.grid-item {
+  width: 100%;
+}
+
+.shop-item {
+  flex-direction: row;
+}
+
+.shop-item__shirt {
+  width: 40%;
```

Comment:
Divying up widths like this can lead to unexpected results. Instead rely on the `flex-shrink` and `flex-grow` to make the `__info` section take up any required width dictated by its content.